integration/src/test/java/org/kie/hacep/PodAsLeaderSnapshotTest.java

```diff
@@ -62,7 +62,7 @@ public void processMessagesAsLeaderAndCreateSnapshotTest() {
                 events = events + eventsRecords.count();
                 int attemptNumber = attempts.incrementAndGet();
                 logger.warn("Attempt number on events topic:{}", attemptNumber);
-                if(attemptNumber == 11){
+                if(attemptNumber == 30){
```

Comment:
Is this change related to the version of hibernate validator? If not please create a separate PR for that and describe why is it useful.

Reply:
Is a workaround to see the build without a problem when jenkins is under load